src/main/java/carpetaddonsnotfound/mixins/network/ClientPlayNetworkHandler_CustomPacketsMixin.java

```diff
@@ -19,30 +18,18 @@
  * handlers should be placed in the ClientNetworkHandler class.
  */
 @Mixin(ClientPlayNetworkHandler.class)
-public abstract class ClientPlayNetworkHandler_CustomPacketsMixin {
-  @Final
-  @Shadow
-  private MinecraftClient client;
+public abstract class ClientPlayNetworkHandler_CustomPacketsMixin extends ClientCommonNetworkHandler {
+  protected ClientPlayNetworkHandler_CustomPacketsMixin(MinecraftClient minecraft,
```

Comment:
Might not need this conmstructor

Reply:
Nevermind I added it because I needed:
`protected final MinecraftClient client;`
from `ClientCommonNetworkHandler`